Client/Application/AppDelegate.swift

```diff
@@ -484,6 +484,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 self.launchFromURL(params)
             }
         }
+
+        guard let extensionDefaults = UserDefaults(suiteName: "group.org.mozilla.ios.Fennec") else { return }
```

Comment:
The context menu part of this patch is good. Why don't we change the bug to reflect that. And file a separate bug for the share extension. I don't want to rush that.